tests/bsim/compile.sh

```diff
@@ -18,10 +18,15 @@ mkdir -p ${WORK_DIR}
 
 source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
 
-run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/audio/compile.sh
-run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/host/compile.sh
-run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/ll/compile.sh
-run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/mesh/compile.sh
-run_in_background ${ZEPHYR_BASE}/tests/bsim/net/compile.sh
+while getops bn flag
+do
+    case "${flag}" in
+    b) run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/audio/compile.sh \
+        run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/host/compile.sh \
+        run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/ll/compile.sh \
+        run_in_background ${ZEPHYR_BASE}/tests/bsim/bluetooth/mesh/compile.sh;;
+    n) run_in_background ${ZEPHYR_BASE}/tests/bsim/net/compile.sh;;
```

Comment:
Maybe better just call tests/bsim/net/compile.sh directly from the workflow,
and move all BT ones in their own subscript (`tests/bsim/bluetooth/compile.sh`) and have that one called from the bt workflow.

Reply:
fixed.